src/slider/README.md

```diff
@@ -55,78 +45,65 @@ export default () => {
 * `StatefulSlider`
 * `StatefulContainer`
 * `StyledRoot`
-* `StyledAxis`
+* `StyledTrack`
+* `StyledInnerTrack`
 * `StyledTick`
 * `StyledTickBar`
 * `StyledThumb`
-* `StyledAxisRange`
+* `StyledInnerThumb`
 
 ## `Slider` API
 
-* `range: Array<Point>` - Required.
-  * All points present on slider axis. First and last point represent min and max
-    value. Each `Point` can be a value itself (for primitive type) or if `Point` is
-    `Object` it should have `value` and `label`. If more than 2 elements are present
-    in array, they represent all ticks on axis and `step` property is ignored.
-* `value: Point | Array<Point>` - Required.
+* `values: Array<number>` - Required.
```

Comment:
What do you think about changing the name of `values` props from plural to single `value`? We use single `value` in select even tho it accepts an array of values for multi-select.

Reply:
Since it's now always an array, I thought plural makes more sense? But I'm fine changing it. Wdyt @chasestarr / @gergelyke?

Reply:
i prefer singular `value`. we will maintain a more similar api to the html element. also, it leaves us open to making a DX improvement of overloading the prop with a single number value in the future.

Reply:
renamed